src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -761,6 +762,20 @@ public virtual string KeyFilePath
             set;
         }
 
+        /// <summary>
+        /// Gets or sets a value for the SSH subsystem to use for the remote connection.
+        /// </summary>
+        [Parameter(ValueFromPipelineByPropertyName = true,
+                   ParameterSetName = PSRemotingBaseCmdlet.SSHHostParameterSet)]
+        public virtual string Subsystem { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value in milliseconds that limits the time allowed for an SSH connection to be established.
+        /// Default timeout value is infinite.
+        /// </summary>
+        [Parameter(ParameterSetName = PSRemotingBaseCmdlet.SSHHostParameterSet)]
+        public virtual int ConnectingTimeout { get; set; } = -1;
```

Comment:
We  directly use the value in a timer as `Timeout.Infinite`. Why not use the name here?
```suggestion
        public virtual int ConnectingTimeout { get; set; } = Timeout.Infinite;
```